DOCS-1605: Adds offset and limit parameters to List all tokens for a single customer reference (#1170)
MultiSafepay/docs: content/api/reference/tokenization-list-all-tokens.md

This API call allows you to list all tokens related to a single customer reference.

> If you have many tokens, you can use the two parameters _limit_ and _offset_ to limit the number of tokens that will be listed.

Reviewer: ```suggestion
To make the response from this API call easier to handle, we automatically paginate results. Use the parameters _limit_ and _offset_ to browse through all tokens under your account.
```

> Example: If the _limit_ is set to 15 and the _offset_ to 0, then 17 tokens will be listed (token 0 to token 16)

Reviewer: ```suggestion
Example: If the _limit_ is set to 15 and the _offset_ to 5, then 15 tokens will be listed (token 5 to token 19)
```

The unique token linked to the customer reference.  

----------------

__limit__ | integer

> The number of tokens listed. If _limit_ is empty, the default value is 10.

Reviewer: ```suggestion
The number of tokens listed. If no _limit_ is specified, the value defaults to 10.
```

----------------

__offset__ | integer

> The position from which the tokens will be listed. If _offset_ is left empty, the default value is 0 (the first token)

Reviewer: ```suggestion
The index from which the tokens will be listed. If no _offset_ is specified, the value defaults to 0 (the first token).
```
